JesusFilm/core
# feat: language filter

# Description

### Issue

Language selection for the searchbar

### Solution

Dropdown
<img width="1049" alt="image" src="https://github.com/user-attachments/assets/9ff3006e-9505-4e98-a6be-1b98aa050219">


# External Changes

n/a

# How to test
* User should be able to select the language button to show language filters dropdown
* User should be able to select a language
* When a language is selected the language should show in the searchbar
* User should be able to remove language by clicking on the language chip
* User should be able to select multiple languages
* When languages are selected they should show results that are one of these languages
* When a user clicks on the searchbar they should see the suggestions dropdown
* If they click in the searchbar after openning the language dropdown the dropdown should not change
* If the select a suggestion the search query should update to that option and apply any language filters only if they haven't already been applied
* User can close dropdown by clicking away
